### ISSUE TITLE
export onnx error

### ISSUE THREAD

**Issue by MZener** [JDAI-CV/CoTNet#14]
<img width="1068" alt="微信截图_20211019093400" src="https://user-images.githubusercontent.com/38679014/137829298-e2cb0c46-d629-43ae-9027-53da312b4f29.png">
cuda11.1 + pytorch1.8 + cupy-cuda111, how to fix it, thanks a lot.

**Comment by QuietWoods** [comment]
根据test_aggregation_zeropad函数将aggregation_zeropad用torch api 实现，以下仅供参考：
`
def custom_aggregation_zeropad(input, weight, kernel_size=3, stride=1, padding=0, dilation=1):
    """
    自定义，用torch api实现
    """
    head_num = 2
    n, c_x, c_w, in_height, in_width = input.shape[0], input.shape[1], weight.shape[2], input.shape[-2], input.shape[-1]
    out_height = int((in_height + 2 * padding - (dilation * (kernel_size - 1) + 1)) / stride + 1)
    out_width = int((in_width + 2 * padding - (dilation * (kernel_size - 1) + 1)) / stride + 1)
    unfold_j = torch.nn.Unfold(kernel_size=kernel_size, dilation=dilation, padding=padding, stride=stride)
    x2 = unfold_j(input).view(n, c_x // c_w, c_w, pow(kernel_size, 2), out_height, out_width)
    y2 = (weight.unsqueeze(2) * x2.unsqueeze(1)).sum(-3).view(n, head_num * c_x, out_height, out_width)
    return y2
`

**Comment by QuietWoods** [comment]
根据test_aggregation_zeropad函数将aggregation_zeropad用torch api 实现，以下仅供参考：
```text
def custom_aggregation_zeropad(input, weight, kernel_size=3, stride=1, padding=0, dilation=1):
    """
    自定义，用torch api实现
    """
    head_num = 2
    n, c_x, c_w, in_height, in_width = input.shape[0], input.shape[1], weight.shape[2], input.shape[-2], input.shape[-1]
    out_height = int((in_height + 2 * padding - (dilation * (kernel_size - 1) + 1)) / stride + 1)
    out_width = int((in_width + 2 * padding - (dilation * (kernel_size - 1) + 1)) / stride + 1)
    unfold_j = torch.nn.Unfold(kernel_size=kernel_size, dilation=dilation, padding=padding, stride=stride)
    x2 = unfold_j(input).view(n, c_x // c_w, c_w, pow(kernel_size, 2), out_height, out_width)
    y2 = (weight.unsqueeze(2) * x2.unsqueeze(1)).sum(-3).view(n, head_num * c_x, out_height, out_width)
    return y2
```

**Comment by YjDai** [comment]
后来这个问题解决了吗?我也是cuda11.1,相同的问题

**Comment by JustinNober** [comment]
俺也是
